script/code-gen-docker.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2046
+
+set -euo pipefail
+
+PROJECT_MODULE="github.com/traefik/hub-crds"
+IMAGE_NAME="kubernetes-codegen:latest"
+CURRENT_DIR="$(pwd)"
+
+echo "Building codegen Docker image..."
+docker build --build-arg KUBE_VERSION=v0.28.4 \
+             --build-arg USER="${USER}" \
+             --build-arg UID="$(id -u)" \
+             --build-arg GID="$(id -g)" \
+             -f "./script/codegen.Dockerfile" \
+             -t "${IMAGE_NAME}" \
+             "."
+
+echo "Generating Traefik clientSet code and DeepCopy code ..."
+docker run --rm \
```

Comment:
Maybe I am wrong but it seems that the code generation is not working. I removed the generated folder, and it is not created again, am I right?